source_code/main_mcu/src/main.c

```diff
@@ -767,3 +773,76 @@ int main(void)
         card_detection_res = smartcard_lowlevel_is_card_plugged();
     }
 }
+
+#if !defined DEBUG_LOG_DISABLED
```

Comment:
should we maybe make it a functionality define such as in https://github.com/mooltipass/minible/blob/master/source_code/main_mcu/src/platform_defines.h#L221 ?
that way we just define it in the platform defines: https://github.com/mooltipass/minible/blob/master/source_code/main_mcu/src/platform_defines.h#L106

Reply:
You mean to move the define of this out of makefile and into the platform_defines.h?

Reply:
I mean have a dedicated define to enable that feature, which would indeed be located in platform_defines.h :)
we could name it STACK_MEASURE_ENABLED

Reply:
:+1: